src/sumOfSquares.js

```diff
@@ -1 +1,4 @@
 'use strict';
+
+module.exports = (x, y) => require('./sum')(require('./square')(x),
+  require('./square')(y));
```

Comment:
compter will understand this, but you should write code that humans can also understand

there are no function names, exports is just a single function (why is called exports then?)
in order to understand what is happening in these two lines, you need to open all files and track exactly what it does

i would not trade short code for understandable code

Reply:
you can take a look at `use` and `awaited imports`, it can create a modular systems, but the use cases are quite narrow

Reply:
Nah that's not the issue, I couldn't get it running because I'd have to also edit test files quite a lot, and I didn't want to do that just to "have some fun"